test/python/visualization/timeline/test_generators.py

```diff
@@ -55,15 +55,17 @@ def test_gen_sched_gate_with_finite_duration(self):
             [-0.5 * self.formatter["box_height.gate"], 0.5 * self.formatter["box_height.gate"]],
         )
         self.assertListEqual(drawing_obj.bits, [self.qubit])
-        ref_meta = {
-            "name": "u3",
-            "label": "n/a",
-            "bits": str(self.qubit.register.name),
-            "t0": 100,
-            "duration": 20,
-            "unitary": "[[1.+0.j 0.-0.j]\n [0.+0.j 1.+0.j]]",
-            "parameters": "0, 0, 0",
-        }
+        # TODO: rewrite this to not use the deprecated `register` property.
+        with self.assertWarns(DeprecationWarning):
```

Comment:
Shouldn't these deprecation warnings have already been issued before? Or why do we need to capture it now? 🙂

Reply:
Yeah, I realized this is because the message changed from what we auto-ignore via QiskitTestCase. I updated that message in another PR that landed last week.

So, in this PR, I removed this change